log/test.log

```diff
@@ -0,0 +1,1020 @@
+Connecting to database specified by database.yml
```

Comment:
Add /log files to .gitignore. db/test.sqlite3 as well. We do not need to commit them.